tests/unit-test/test_cache.c

```diff
@@ -8,32 +8,56 @@
 
 #include "tests/test_define.h"
 #include "utils/cache/cache.h"
+#include "uuid/uuid.h"
+
+char test_uuid[UUID_STR_LEN] = {};
 
 void test_cache_del(void) {
-  const char* key = TRYTES_81_1;
-  cache_del(key);
+  const char* key = strdup(test_uuid);
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_del(key));
+  free(key);
 }
 
 void test_cache_get(void) {
-  const char* key = TRYTES_81_1;
-  char res[TRYTES_2673_LEN + 1] = {0};
-  cache_get(key, res);
-  res[TRYTES_2673_LEN] = '\0';
-  TEST_ASSERT_EQUAL_STRING(res, TRYTES_2673_1);
+  const char* key = strdup(test_uuid);
+  char res[strlen(CACHE_VALUE) + 1];
+
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_get(key, res));
+  TEST_ASSERT_EQUAL_STRING(CACHE_VALUE, res);
+  free(key);
 }
 
 void test_cache_set(void) {
-  const char* key = TRYTES_81_1;
-  const char* value = TRYTES_2673_1;
-  cache_set(key, value);
+  char* key = strdup(test_uuid);
+  const char* value = CACHE_VALUE;
+
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_set(key, strlen(key), value, strlen(value), 0));
+  free(key);
+}
+
+void test_cache_timeout(void) {
+  char* key = strdup(test_uuid);
+  const char* value = CACHE_VALUE;
+  char res[strlen(CACHE_VALUE) + 1];
+  const int timeout = 2;
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_set(key, strlen(key), value, strlen(value), timeout));
+  TEST_ASSERT_EQUAL_INT(SC_OK, cache_get(key, res));
+  sleep(timeout + 3);
```

Comment:
Why timeout+3?

Reply:
we need a certain time to ensure the data has been removed from the database